k8s/cloudbuild.yaml

```diff
@@ -18,7 +18,7 @@ steps:
       - --build-arg
       - NEXT_PUBLIC_GA_TRACKING_ID=G-K9HQ2LE04G
       - --build-arg
-      - NEXT_PUBLIC_DUST_CLIENT_FACING_URL=${_DUST_CLIENT_FACING_URL}
+      - NEXT_PUBLIC_DUST_CLIENT_FACING_URL=${_DUST_CLIENT_FACING_URL:-}
```

Comment:
Is this one expected?

Reply:
Yeah, default to an empty string if not provided

Reply:
Otherwise we need to do some crazy bash logic to not pass the build-arg

Reply:
Something like:
```
      - $([[ -n "${_DUST_CLIENT_FACING_URL}" ]] && echo "--build-arg NEXT_PUBLIC_DUST_CLIENT_FACING_URL=${_DUST_CLIENT_FACING_URL}")
```
and I'm afraid the `[[` syntax is gonna break again